# Margin ratio of grid map
grid_map_margin_ratio: 0.25

# Resolution of grid map [m]
grid_map_resolution: 0.02

# Height scale of grid map
grid_map_height_scale: 1.0

# Theta threshold for slicing SE2 sample [deg]
slice_se2_theta_thre: 2.5

# Z threshold for slicing R3 sample [m]
slice_r3_z_thre: 0.025

# Z threshold for slicing SE3 sample [m]
slice_se3_z_thre: 0.1

# Theta threshold for slicing SE3 sample [deg]
slice_se3_theta_thre: 20

# ROS bag path of sample set for evaluation
eval_bag_path: "" # Set by rosparam since it contains ROS package path

# SVM threshold list for evaluation
eval_svm_thre_list: [-0.2, -0.15, -0.1, -0.05, 0.0]

# Threshold list of distaice ratio for evaluation by one-class nearest neighbor
ocnn_dist_ratio_thre_list: [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]

# K (i.e., number of nearest samples) list for evaluation by k-nearest neighbor
knn_K_list: [1, 3, 5, 7, 9]

# Height of xy plane marker
svm_thre: -0.1

# SVM gamma (smoothness. smaller is smoother)
svm_gamma: 30

# SVM nu (outliers ratio)
svm_nu: 0.05
